package rest.felix.back.todo.exception;

import rest.felix.back.common.exception.throwable.notFound.ResourceNotFoundException;

public class TodoNotFoundException extends ResourceNotFoundException {}
